doc/user/content/concepts/clusters.md

```diff
@@ -34,8 +34,12 @@ You can resize a cluster to respond to changes in your workload.
 
 {{< note >}}
 
-Resizing incurs downtime, as it requires all objects in the cluster to hydrate.
-In the future, it will be possible to resize clusters with no downtime {{% gh 20010 %}}.
+Resizing incurs downtime as it requires all objects in the cluster to hydrate.
```

Comment:
I thought you meant mentioning this in the `CREATE CLUSTER` page, not in the concepts page, @kay-kim. I think this is too detailed for this page. I'll push a commit that softens the mention here and adds a mention to `CREATE CLUSTER`.